docs/manuals/cli/run.md

```diff
@@ -0,0 +1,110 @@
+# Run application in Taipy CLI
+
+Assume that you have a "main.py" file containing your Taipy application.
+To start the Taipy application, you can run:
+
+```console
+$ python main.py
+```
+
+However, when working with command-line arguments, Taipy supported arguments can be confused with
+arguments of other libraries or your application's arguments. To avoid this confusion, Taipy CLI
+provides a run command to run your Taipy application, supporting both Taipy supported and unsupported arguments.
```

Comment:
What do you think of the following suggestion?

```suggestion
!!! note

    An alternative way to run your application is to use the Python command directly.
    
    ```console
    $ python main.py
    ```
    
    However, we recommend using the `taipy run` command, which is more robust with command line arguments. As explained in the next section, it avoids any argument conflict in case another library or your interpreter uses the same argument as Taipy.

```

Reply:
This makes sense